### ISSUE TITLE
Broken link in  Docs/Getting started/Email Piping.rst

### ISSUE THREAD

**Issue by ChrisJon02** [osTicket/docs#155]
How can I fix that broken link? Because I can't find the related blog by dave.

**Comment by JediKev** [comment]
@ChrisJon02 

Can you provide more info on what exactly you mean? You can always submit the change here and we can make it; we just need as much info as possible.

Cheers.

**Comment by ChrisJon02** [comment]
@JediKev  Yes, I'm talking about that blog post from Dave link which is showing a 404 error

**Comment by JediKev** [comment]
@ChrisJon02 

I forgot to update this thread. This was resolved with the following changes:
- https://github.com/osTicket/docs/pull/160

Here is a link to the original post if you need it:
- https://web.archive.org/web/20210211003622/https://blog.absolutedisaster.co.uk/osticket-plesk-9-postfix-pipe-mail-to-a-progr/

Cheers.
